README.md

```diff
@@ -13,6 +13,41 @@ Encryption](https://en.wikipedia.org/wiki/End-to-end_encryption)) for
 
 ## Usage
 
+1. Install in your project:
+
+    ```
+    npm install --save @matrix-org/matrix-sdk-crypto-wasm
```

Comment:
Can we stick to `yarn` or `npm`, but not have both?

Reply:
The problem is that you have to use the tool that the project you're building wants -- and `npm` is the lowest common denominator: people who use `yarn` can be expected to know how to translate an `npm` commandline into `yarn`, but `npm` users do not necessary know about `yarn`. Hence there's some sort of loose convention that "install this library" instructions use `npm`.

I can include yarn instructions as well.